### DISCUSSION TITLE
SSH_FOLLOW_SYMLINKS missing

### DISCUSSION THREAD

**Discussion by Tuharsky** [ltsp/ltsp#468] [Q&A]
Hi, guys,

please, is there a possibility to make symlinks work like they used to with old LTSP? SSH_FOLLOW_SYMLINKS directive functionality is missing in ltsp.conf AFAIK...

**Comment by alkisg** [comment]
Hi, have a look at issue #114.

**Reply by Tuharsky** [reply]
Well, I have already, but I'm afraid the solution is not clear for me. There is a mention to use SSH_OPTIONS but I cannot find proper parameters and/or syntax that should be used for this goal in manual page
https://ltsp.org/man/ltsp.conf/

**Comment by alkisg** [comment]
LTSP5 didn't use SSH_FOLLOW_SYMLINKS by default because it breaks some applications and possibly even the whole session. The same is true for the new LTSP as well.
If you really want to enable it, you put the following in ltsp.conf, then you run `ltsp initrd`:

SSH_OPTIONS="-o follow_symlinks"

**Reply by Tuharsky** [reply]
Right to the point :-)
I put it in [common] section and it works, thank You. Might be valuable to mention the parameter on the man page https://ltsp.org/man/ltsp.conf/

Have a nice day, Alkis.
